fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -171,6 +171,17 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
             } else {
                 throw new SemanticException("mode argument must be numerical type");
             }
+
+            Expr windowArg = functionCallExpr.getChild(0);
+            if (windowArg instanceof IntLiteral) {
+                IntLiteral windowIntLiteral = (IntLiteral) windowArg;
+                long windowValue = windowIntLiteral.getValue();
+                if (windowValue < 0) {
+                    throw new SemanticException("window argument must >= 0");
+                }
+            } else {
+                throw new SemanticException("window argument must be numerical type");
```

Comment:
UT should cover this Exception

Reply:
The Exception: window argument must >= 0 and window argument must be numerical type both need to cover